smithy-aws-traits/src/main/java/software/amazon/smithy/aws/traits/tagging/TaggableResourceValidator.java

```diff
@@ -87,7 +87,6 @@ private List<ValidationEvent> validateResource(
         //    list or tag keys input or output member
         // 2. Tagging via APIs specified in the @taggable trait which are validated
         //    through the tag property, and must be resource instance operations
-        //Caution: avoid short circuiting behavior.
```

Comment:
This comment is misleading, you only want to avoid short circuiting if you are expecting a side effect, that's not the case here.